routes/userProfile.js

```diff
@@ -0,0 +1,80 @@
+const router = require('express').Router()
+const db = require('../db')
+const authorization = require('../middleware/authorization')
+const calculateAge = require('../utils/calculateAge')
+const Constants = require('../utils/constant')
+
+const { months, validUUID } = Constants
+
+// send back current logged in user after token validation
+router.get('/loggedInUser', authorization, async (req, res) => {
+  try {
+    const user = await db.query('SELECT * FROM user_profile WHERE user_id=$1', [
+      req.loggedInUserId,
+    ])
+    const { first_name, last_name } = user.rows[0]
+    return res.json({
+      id: req.loggedInUserId,
+      firstName: first_name,
+      lastName: last_name,
+    })
+  } catch (err) {
+    console.error(err.message)
```

Comment:
im not sure if the message contains it but consider also logging the inputs that you care about. the message may already contain this but something to keep note of in the future in the case that the default error messages are not robust enough